OpenSim/Common/ModelDisplayHints.h

```diff
@@ -102,6 +102,9 @@ class OSIMCOMMON_API ModelDisplayHints : public Object {
     OpenSim_DECLARE_PROPERTY(show_debug_geometry, bool,
         "Flag to indicate whether or not to show debug geometry, default to false.");
 
+    OpenSim_DECLARE_PROPERTY(decorations_scale_factor, double,
+        "Scale factor to apply to decorations that have no inherent size but should scale with the overall size of the model (e.g. Marker, ContactHalfSpace); defaults to 1.");
```

Comment:
Thanks for the feedback @aseth1 As indicated at the top of the PR, this is intended to handle models of arbitrary size. Mesh files have built-in/assumed units, same with any real/physical geometry that assumes model length units, some pieces of Geometry however doesn't have any physical meaning (e.g. a Marker)  even if it had a radius you'll have to place it in the xml file. Having multiple numbers that you need to change (one for Markers, another for muscles, a third for Frame etc. doesn't seem to "scale").
There's no infinite plane or plane primitive that's why I used a brick, the API visualizer uses an opengl program to draw, and the Three.js uses a "finite" size plane but these used for ground do not generalize if you have multiple ContactHalfSpaces for example.